include/experimental/group/gemm/impl/int4_dequantize_xmx_xe.hpp

```diff
@@ -478,512 +499,48 @@ class gemm_t<
                 auto scale_vec
                         = scale.reg.xetla_select<scale_t::block_size_x, 1>(
                                 scale_block_id * scale_t::block_size_x);
-                auto zero_pt_vec
-                        = zero_pt.reg
-                                  .xetla_select<zero_pt_t::block_size_x, 1>(
-                                          scale_block_id
-                                          * zero_pt_t::block_size_x)
-                                  .xetla_format<uint8_t>();
 
                 auto dst_blk
                         = matB_acc.reg.xetla_select<matB_acc_t::block_elems, 1>(
                                 block_id * matB_acc_t::block_elems);
 
-                xetla_vector<uint8_t, block_size_x_b> zero_pt_sub;
                 //2: int8 includes 2 4bits data.
-                zero_pt_sub.xetla_select<block_size_x_b / 2, 2>(0)
-                        = zero_pt_vec & 0x0f;
-                zero_pt_sub.xetla_select<block_size_x_b / 2, 2>(1)
-                        = zero_pt_vec >> 4;
-
-                xetla_vector<uint8_t, block_size_x_b * block_size_y_b>
-                        zero_pt_blk;
-#pragma unroll
-                for (int row = 0; row < block_size_y_b; row++) {
-                    zero_pt_blk
-                            .xetla_select<block_size_x_b, 1>(
-                                    row * block_size_x_b)
-                            .xetla_format<int8_t>()
-                            = zero_pt_sub.xetla_format<int8_t>() + int8_t(1);
-                }
-
                 xetla_vector<uint8_t, block_size_x_b * block_size_y_b> cvt_blk;
                 cvt_blk.xetla_select<matB_t::block_elems, 2>(0)
                         = matB_blk & 0x0f;
                 cvt_blk.xetla_select<matB_t::block_elems, 2>(1) = matB_blk >> 4;
```

Comment:
seems that we unpack a int4-value and move it 4bit to a uint8_t's low-4-bit postion.
so it's not the `S4_CLIP` definition in Jblas(we keep the high-4-bit position in Jblas).
I guess this is what S4_ASYM definition in inc